### PR TITLE
Added a test to avoid method on null

### PR DESCRIPTION
**Pull request by kgf**
Cuz that never turns out well.

### CODE REVIEW

File: app/views/analytics/system_activity_logs.html.erb [hunk 1]
```diff
@@ -15,7 +15,7 @@
   <tbody>
     <% @logs.each do |log| %>
       <tr>
-        <td><%= log.user.fullname %></td>
+        <td><%= log.user.present? ? log.user.fullname : "Unknown User" %></td>
```
Comment (gasparyn) [inline app/views/analytics/system_activity_logs.html.erb]: also
```
<td><%= log&.user&.fullname || "Unknown User" %></td>
```

Comment (kgf) [reply app/views/analytics/system_activity_logs.html.erb]: Well, `log` can't be null, so `<%= log.user&.fullname || "Unknown User" %>` would also be acceptable ?

Comment (gasparyn) [reply app/views/analytics/system_activity_logs.html.erb]: even better. 👍

**Comment by gasparyn** [comment]
looks like we need to rebase with development too

**Review by gasparyn — CHANGES_REQUESTED** [review]
rebase needed

**Review by gasparyn — APPROVED** [review]

**Pull request merged**